File: Modul2_C/PRAK305-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main(){
    int input;
    scanf("%d", &input);

    int hari = 0, jam = 0, menit = 0, detik = 0;

    hari = input / 86400;

    jam = (input % 86400)/3600;

    menit = (input % 3600)/60;

    detik = input % 60;

    if(hari == 0){
        ptintf("%.2d:%.2d:%.2d", jam, menit, detik);
    } else{
        printf("%d hari %.2d:%.2d:%.2d", hari, jam, menit, detik);
    }

    return 0;
}

File: Modul2_Pyton/PRAK305-2510817320010-PutriDeissytaHerliyana.py
inputdetik = int(input())

hari = inputdetik // 86400
jam = (inputdetik % 3600) // 3600
menit = (inputdetik % 3600) // 60
detik = inputdetik % 60

if hari == 0:
 print(f"{jam:02d}:{menit:02d}:{detik:02d}")
else:
    print(f"{hari} hari {jam:02d}:{menit:02d}:{detik:02d}")